ui/src/testing/mock.rs

```diff
@@ -60,8 +60,10 @@ pub fn generate_mock() -> State {
     State {
         ui: UI {
             current_call: None,
+            current_debug_logger: None,
             overlays: vec![],
             popout_player: false,
+            debug_logger: false,
```

Comment:
i think this field is redundant and that `current_debug_logger.is_some()` is equivalent to `debug_logger`